packages/studio-base/src/panels/RawMessages/index.tsx

```diff
@@ -156,7 +157,7 @@ function RawMessages(props: Props) {
   }, [datatypes, topic, topicRosPath]);
 
   // When expandAll is unset, we'll use expandedFields to get expanded info
-  const [expandAll, setExpandAll] = useState<boolean | undefined>(false);
+  const [expandAll, setExpandAll] = useState<boolean | undefined>(props.defaultExpandAll ?? false);
```

Comment:
No longer can be undefined
```suggestion
  const [expandAll, setExpandAll] = useState(props.defaultExpandAll ?? false);
```

Reply:
There is code elsewhere in the file that sets expand all to undefined. That code could set it to false instead.